tests/schemas/en/test_language.json

```diff
@@ -17,13 +17,19 @@
             "type": "string"
         }
     ],
+    "submission": {
+        "button": "Submission text",
+        "guidance": "Submission guidance",
+        "title": "Submission title",
+        "warning": "Submission warning"
+    },
```

Comment:
This schema is slightly different to the one in runner - it's missing the hub enabled property and there are some minor formatting differences.